__tests__/comments.tests.js

```diff
@@ -0,0 +1,71 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+const sorted = require('jest-sorted')
+
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe('/api/articles/:article_id/comments', () => {
+    it('404: article not found', async () => {
+        try {
+            const result = await request(app)
+            .get('/api/articles/34234234/comments')
+        } catch(error) {
+            expect(error).toMatchObject({
+                status: 404, 
+                msg: "No article found"
+            })
+        }
+    });
+    it('400: not a number', async () => {
+        try{
+            const { body } = await request(app)
+            .get('/api/articles/e/comments')
+        } catch(error) {
+            expect(error).toMatchObject({
+                status: 400,
+                msg: "Wrong data type"
+            })
+        }
+    });
+    it('should return comments', async () => {
+        try {
+            const { body } = await request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            body.forEach((comment) => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    body: expect.any(String),
+                    article_id: expect.any(Number),
+                    author: expect.any(String),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String)
+                })
+            })
```

Comment:
The body shouldnt be the array of comments, it should instead be an object with a key of comments with the value of the array of comments. Because we have an array here as well we want to make sure we fail successfully if given an empty array so before this forEach we need to assert that there is at least 1 element in the array so using a matcher similar to .toBeGreaterThan would be really helpful.